utils/subxt/codegen/bin/main.rs

```diff
@@ -28,22 +30,40 @@ pub struct Cli {
 	pub para_host: String,
 	#[clap(long, default_value = "9188")]
 	pub para_port: String,
+	/// Path to parachain runtime wasm blob
+	#[clap(long)]
+	pub parachain_wasm: Option<PathBuf>,
```

Comment:
Why using option?

Reply:
A user can still choose to fetch it from a live node, fetching the metadata from wasm blob is optional.